lib/jsonld-signatures/suites/ed255192018/ed25519.ts

```diff
@@ -31,15 +24,18 @@ const api = {
    * @returns {object} The object with the public and private key material.
    */
   async generateKeyPairFromSeed(seedBytes) {
-    const privateKey = await createPrivateKey({
-      // node is more than happy to create a new private key using a DER
-      key: _privateKeyDerEncode({ seedBytes }),
-      format: 'der',
-      type: 'pkcs8',
-    });
+    const privateKey: forge.pki.PrivateKey = forgePrivateKey(
+      _privateKeyDerEncode({seedBytes}),
+    );
+
     // this expects either a PEM encoded key or a node privateKeyObject
-    const publicKey = await createPublicKey(privateKey);
-    const publicKeyBuffer = publicKey.export(publicKeyEncoding);
+    const publicKey: forge.pki.PublicKey =
```

Comment:
can't we import  `forge.pki. PublicKey ` as `ForgePublicKey`?

Reply:
Refactored